orchestrator/relayer/batch_relaying.go

```diff
@@ -242,16 +245,22 @@ func (s *peggyRelayer) IsBatchProfitable(
 	}
 
 	usdTokenPriceDec := decimal.NewFromFloat(usdTokenPrice)
-	// decimals (uint8) can be safely casted into int32 because the max uint8 is 255 and the max int32 is 2147483647
+	// Decimals (uint8) can be safely casted into int32 because the max uint8 is 255 and the max int32 is 2147483647.
 	totalFeeInUSDDec := decimal.NewFromBigInt(totalBatchFees, -int32(decimals)).Mul(usdTokenPriceDec)
 
+	// Simplified: totalFee > (gasCost * profitMultiplier).
+	isProfitable := totalFeeInUSDDec.GreaterThan(gasCostInUSDDec.Mul(decimal.NewFromFloat(profitMultiplier)))
```

Comment:
shouldn't this be >=?

Reply:
Right, changed it to `GreaterThanOrEqual`